frontend/javascripts/oxalis/view/action-bar/save_button.js

```diff
@@ -15,19 +18,33 @@ type OwnProps = {|
 type StateProps = {|
   progressInfo: ProgressInfo,
   isBusyInfo: IsBusyInfo,
+  oldestUnsavedTimestamp: ?number,
 |};
 type Props = {| ...OwnProps, ...StateProps |};
 
 type State = {
   isStateSaved: boolean,
+  currentTimestamp: number,
 };
 
 const SAVE_POLLING_INTERVAL = 1000;
+const UNSAVED_WARNING_THRESHOLD = 5 * 1000;
+
+const reportUnsavedDurationThresholdExceeded = _.once(() => {
```

Comment:
I'm thinking whether we want to send these periodically (throttled to a larger value like 10 minutes)? But we can also wait and look how often these happen and whether a more detailed logging would help :)